### DISCUSSION TITLE
food/mocktails/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/food#9] [General]
# food/mocktails/

https://shanenull.com/food/mocktails/

**Comment by shane0** [comment]
# pina colada


Just a thought, try it like you're making an old school egg cream.

1 8 floz can pineapple juice

4 fl oz white sugar or sweetner of your choice

4oz coconut milk from a can, fat blended with the liquid

4oz chilled soda water

My suggestion would be to reduce the pineapple juice to a syrup by reducing it by half boiling it on a stove and adding sugar. Combine juice and sugar in a pan, heat and stir to make sure the sugar is dissolved. Take it off the heat and cool completely before using.

Add 1 shot of pineapple juice and 4oz coconut milk (I'd do the canned kind, but mixed up, for the texture) in a shaker with ice. Shake until cold and frothy, then pour into acollins glass or pint glass with 4oz cold soda water. Garnish with a metal straw and small wedge of pineapple

I'm just spitballing here. If you do this, lmk how it turns out! I have to try it myself now...


---

For the coconut pineapple goodness:
1 22-oz, container of coconut cream

1 12-oz. can of frozen pineapple juice concentrate

Juice of 1 – 2 limes or lemons

1/2 cup water

1 14-oz. can light coconut milk

For each drink:
1 oz. coconut rum (optional!)

1 to 1 1/2 oz. of the coconut pineapple goodness (see above)

Cold club soda or other sparkling water

Instructions
Blend together the ingredients for the coconut pineapple goodness, put in a covered plastic container and put in the freezer until it’s good and cold.

To make the drinks: Pour a shot of the coconut pineapple goodness in a glass. Fill the glass with your choice of fizzy water. Stir and sip … is it sweet enough? If not add a little more coconut pineapple goodness. Garnish with pineapple if you like and get yourself a straw for slurping up this refreshing beverage. Enjoy!

**Comment by shane0** [comment]
- https://tuxedono2.com/ingredients/peychaud-s-bitters-cocktail-recipes

**Comment by shane0** [comment]
https://www.seedlipdrinks.com/en-us/shop/seedlip-spice-94/

**Reply by shane0** [reply]
# sugar free syrup

- earl grey simple syrup
- lemon juice

## the duke

- found at nordstrom bazille
- add seedlip 94
- above sugar free simple syrup

**Reply by shane0** [reply]
https://www.seedlipdrinks.com/en-us/recipes/

**Comment by shane0** [comment]
# 24/7

24/7 14.00 / 80 cal
seedlip grove 42, demerara, owen's nitro-infused espresso, fever tree tonic, lemon add aviation gin 19.00 / 180 cal
JUICES 5.75 / 110 - 150 cal
